src/qt/locale/bitcoin_sv.ts

```diff
@@ -1156,72 +1152,72 @@ Detta betyder att en avgift på minst %2 krävs.</translation>
         <location filename="../forms/diagnosticsdialog.ui" line="+14"/>
         <location line="+210"/>
         <source>Diagnostics</source>
-        <translation type="unfinished">Diagnostik</translation>
+        <translation>Diagnostik</translation>
     </message>
     <message>
         <location line="-157"/>
         <source>Verify CPID is in Neural Network</source>
-        <translation type="unfinished"></translation>
+        <translation>Verifiera att CPID finns i nätverket</translation>
     </message>
     <message>
         <location line="+51"/>
         <source>Verify BOINC path</source>
-        <translation type="unfinished"></translation>
+        <translation>Verifiera BOINC sökväg</translation>
```

Comment:
BOINC-sökväg. I think I missed a couple of these higher up :)

Reply:
Well there are more phrases to translate. Will do some more in a month or two.